eox_tenant/test/test_signals.py

```diff
@@ -283,3 +292,95 @@ def test_udpate_settings_with_existing_dict(self, _get_config_mock):
 
         self.assertEqual(settings.TEST_DICT_OVERRIDE_TEST.get("key1"), "Some Value")
         self.assertEqual(settings.TEST_DICT_OVERRIDE_TEST.get("key2"), "My value")
+
+
+class CelerySignalCLISyncTests(TestCase):
+    """
+    Testing the celery signals generated outside of a request in a sync process.
+    """
+    def setUp(self):
+        """ setup """
+        for number in range(3):
+            Site.objects.create(  # pylint: disable=no-member
+                domain="tenant{number}.com".format(number=number),
+                name="tenant{number}.com".format(number=number)
+            )
+
+    def test_get_host_from_siteid(self):
```

Comment:
here you don't test a signal or receiver, so  imo this deserves an own testing class